lib/info-plist.js

```diff
@@ -5,10 +5,13 @@ const plist = require('simple-plist');
 const appleDevices = ['iPhone', 'iPad', 'AppleTV', 'AppleWatch'];
 const objectFromEntries = (x) => Array.from(x, (k) => ({ [k]: [] })); // ES7 is not yet here
 
-function fix (file, options, emit) {
-  const { appdir, bundleid, forceFamily, allowHttp } = options;
-  if (!file || !appdir) {
-    throw new Error('Invalid parameters for fixPlist');
+function adjustInfoPlist (file, options, emit) {
+  let { bundleid, forceFamily, allowHttp } = options;
+  if (!file) {
+    throw new Error('Invalid parameters for adjustInfoPlist');
+  }
+  if (file.indexOf('.appex/') != -1) {
+    bundleid += '.share';
```

Comment:
network extensions may use: `.network-extension` so this is just a partial fix.

Probably the way to go is to check for the NSExtension array and see which is the `NSExtensionPrincipalClass` used for each appex

Reply:
`.SiriExtension` for NSExtension.NSExtensionPointIdentifier == com.apple.intents-service

Reply:
`com.apple.networkextension.packet-tunnel` -> network-extension

Reply:
shouldn't this happen only if `bundleId` is defined? AFAIK overriding the bundle id is optional

Reply:
F

Reply:
LOL for the single letter contributor